xml/System.Windows.Forms/TaskDialogPage.xml

```diff
@@ -280,7 +410,7 @@
         <ReturnType>System.EventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Occurs when the user presses F1 while the task dialog has focus, or when the user clicks the <see cref="P:System.Windows.Forms.TaskDialogButton.Help" /> button.</summary>
```

Comment:
Same question here, can we use `<kbd>`?

Reply:
See https://github.com/dotnet/dotnet-api-docs/pull/4706#discussion_r477854624